src/components/structures/RoomView.js

```diff
@@ -1728,8 +1731,8 @@ module.exports = React.createClass({
             <TimelinePanel ref={this._gatherTimelinePanelRef}
                 timelineSet={this.state.room.getUnfilteredTimelineSet()}
                 showReadReceipts={!UserSettingsStore.getSyncedSetting('hideReadReceipts', false)}
-                manageReadReceipts={true}
-                manageReadMarkers={true}
+                manageReadReceipts={!this.state.isPeeking}
```

Comment:
doesn't this also mean that you won't be able to see read receipts whilst peeking? I know the additional prop showReadReceipt is there but I think it requires manageReadReceipts=true

Reply:
I'll double check, but in my testing I don't remember it being failing.

Reply:
can confirm: read receipts still visible for other people.